src/test/kotlin/com/fasterxml/jackson/module/kotlin/test/failing/Github138.kt

```diff
@@ -23,11 +24,14 @@ class TestGithub138 {
         )
 
     @Test
-    @Ignore("Not sure the cause of this yet...")
     fun testDeserProblem() {
         val xml = """<sms Phone="435242423412" Id="43234324">Lorem ipsum</sms>"""
         val xmlMapper = XmlMapper().registerKotlinModule()
-        val sms = xmlMapper.readValue<Sms>(xml)
-
+        try {
+            val jsms = xmlMapper.readValue<Sms>(xml)
+            fail("GitHub #138 has been fixed!")
+        } catch (e: InvalidDefinitionException) {
+            // Remove this try/catch and the `fail()` call above when this issue is fixed
+        }
```

Comment:
An example of the new style test—the call to `readValue()` throws `InvalidDefinitionException` which is caught in the current state of the world.  If that ever changes, we'll be greeted with a test failure with the reasonably-clear message `GitHub #138 has been fixed!`.